man/org.freedesktop.systemd1.xml

```diff
@@ -3148,7 +3148,7 @@ node /org/freedesktop/systemd1/unit/avahi_2ddaemon_2eservice {
       readonly i SyslogLevel = ...;
       @org.freedesktop.DBus.Property.EmitsChangedSignal("const")
       readonly i SyslogFacility = ...;
-      @org.freedesktop.DBus.Property.EmitsChangedSignal("const")
+      @org.freedesktop.DBus.Property.EmitsChangedSignal("false")
```

Comment:
i don't think we should mangle this in here. just leave the log level reported with this property as is, but add a new bool property that indicates the debug start mode. (which is then 1:1 reflected in the passed env var)